src/components/content/style.css

```diff
@@ -0,0 +1,47 @@
+@value red, turquoise from "../variables/colors.css";
+
+.content {
+
+}
+
+.content a {
+  color: red;
+  white-space: nowrap;
+}
+
+.content a:hover {
+  color: turquoise;
+}
+
+.p {
+  composes: defaultFont from "../fonts/style.css";
+  margin: 1em 0;
+  line-height: 1.35;
+}
+
+.h1 {
```

Comment:
Exactly my thinking. If we go down this route we can remove the content component. It will also allow us to declare the `api` for each of these to restrict how they can be used. This is good from a design perspective because we can accurately test these in their various states.